typescript/sdk/src/utils/gnosisSafe.js

```diff
@@ -51,6 +51,11 @@ const chainOverrides = {
     multiSend: '0x0dFcccB95225ffB03c6FBB2559B530C2B7C8A912',
     multiSendCallOnly: '0xf220D3b4DFb23C4ade8C88E526C1353AbAcbC38F',
   },
+  // berachain
+  80094: {
+    multiSend: '0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761',
+    multiSendCallOnly: '0x40A2aCCbd92BCA938b02010E17A5b8929b49130D',
```

Comment:
we use different multisend contracts for reads and writes?

Reply:
this is just taken from the safe-deployments repo

https://github.com/safe-global/safe-deployments/blob/9d89b7ae83b84e02209d95e436ea230f0b10b307/src/assets/v1.3.0/multi_send.json#L7
https://github.com/safe-global/safe-deployments/blob/9d89b7ae83b84e02209d95e436ea230f0b10b307/src/assets/v1.3.0/multi_send_call_only.json#L7

and cross-referenced with the deployed contracts on berachain

Reply:
multisend vs multisendcallonly is not read vs write, it's whether there are nested delegate calls or not